bindings/wysiwyg-ffi/src/ffi_composer_model.rs

```diff
@@ -251,4 +251,9 @@ impl ComposerModel {
     pub fn get_link_action(self: &Arc<Self>) -> LinkAction {
         self.inner.lock().unwrap().get_link_action().into()
     }
+
+    /// Force a panic for test purposes
+    pub fn force_panic(self: &Arc<Self>) {
+        panic!("This should only happen in tests.");
+    }
```

Comment:
I think I'd rather see that not committed to the repo. We can force crash easily with some very invalid `set_content_from_html` usage. e.g. I use "<//strong>" but I guess a lot of other things can work.

Reply:
The reason I added this was so that we could make it explicit and guarantee the crash would happen in tests (whereas something like `set_content_from_html("<//strong>")` is a bug that we could expect to be fixed later).

Would it help if I put it behind a conditional compilation so that there's no risk of this getting into production? E.g.


    /// Force a panic for test purposes (no-op in release builds)
    #[cfg(not(debug_assertions))]
    pub fn debug_panic(self: &Arc<Self>) {
        // No-op
    }

    /// Force a panic for test purposes
    #[cfg(debug_assertions)]
    pub fn debug_panic(self: &Arc<Self>) {
        panic!("This should only happen in tests.");
    }


Ideally we could also exclude it from the binding interface too but I couldn't work a way to do this.

Reply:
Yeah I don't think you could translate Rust cfg easily to have this magically disappear on production build. 
I guess this is fine then.